testing/regress/ecl/memcachedtest.ecl

```diff
@@ -17,6 +17,7 @@
 
 //class=embedded
 //class=3rdparty
+//nothor
```

Comment:
I've decided to go down the individual file route and use //nothor on the new file for error testing. The reason being that using platform conditional code in this manner requires me to produce a separate xml key file in testing/regress/ecl/roxie/ anyhow. Since https://track.hpccsystems.com/browse/HPCC-14963 to complete the tests now also exists this may make more sense.